tests/testAutopas/tests/tuning/tuningStrategy/LiveInfoTest.cpp

```diff
@@ -9,9 +9,200 @@
 #include "autopas/AutoPas.h"
 #include "autopas/tuning/tuningStrategy/LiveInfo.h"
 #include "autopas/utils/ArrayMath.h"
+#include "autopas/utils/ExceptionHandler.h"
 #include "autopas/utils/ParticleBinStructure.h"
 #include "testingHelpers/commonTypedefs.h"
 
+/**
+ * Tests that the parametrized constructor creates the correct info mappings.
+ */
+TEST_F(LiveInfoTest, ParametrizedConstructorCreatesCorrectInfo) {
+  constexpr double epsilon = 1e-10;
+  const std::vector<std::tuple<size_t, size_t, double, double, double, double, double, size_t, size_t, size_t, size_t,
+                               size_t, size_t, size_t, size_t, size_t, size_t, double, double, double, size_t, double,
+                               double, size_t, size_t, size_t, size_t, size_t, double, double, double>>
+      parameters = {
+          {10, 5,   2.0, 0.1,  10.0, 10.0, 10.0, 64, 4, 20, 8, 2, 1,   3,   2,   1,
+           3,  2.0, 0.5, 0.25, 100,  0.8,  0.05, 4,  1, 2,  1, 3, 2.0, 0.3, 0.15},
+          {50, 25,  3.0, 0.2, 20.0, 20.0, 20.0, 128, 8, 30, 16, 4, 2,   6,   4,  2,
+           6,  3.0, 1.0, 0.5, 200,  1.6,  0.1,  8,   2, 4,  2,  6, 3.0, 0.6, 0.3},
+          {25, 8,   0.750, 0.15, 15.0, 15.0, 15.0, 96, 6, 25, 12, 3, 1,   4,    3,  2,
+           4,  1.5, 0.75,  0.35, 150,  1.2,  0.08, 6,  1, 3,  1,  4, 1.5, 0.45, 0.2},
+          // Constructor with the lowest allowed values
+          {1, 0,       epsilon, 0.0, epsilon, epsilon, epsilon, 1, 1, 1, 1, 0, 0,       0,   0,  0,
+           0, epsilon, 0.0,     0.0, 1,       epsilon, 0.0,     1, 0, 0, 0, 0, epsilon, 0.0, 0.0},
```

Comment:
I don't think `epsilon` is the lowest positive number. But also, you tend to allow `0.0` in your checks. So maybe replace `epsilon` with `0.0`. I'm not totally sure of the purpose of this test, so check what I am saying.